roles/nfs-server/tasks/main.yml

```diff
@@ -1,18 +1,30 @@
 ---
-- name: install nfs server
-  action: apt pkg=nfs-kernel-server state=present update_cache=yes
+- name: install mdadm
+  apt: update_cache=yes cache_valid_time=600 name=mdadm state=latest
+  sudo: yes
   tags:
-    -nfs
+    - nfs
+
+- name: run the lsi-cards script
+  script: lsi-cards {{ ansible_fqdn }} /var/lib/docker/export creates=/etc/{{ ansible_fqdn }}.raid-setup
```

Comment:
I actually set the lsi-cards on the main box at `/var/lib/docker`, making sure to do it *before* I install Docker.